index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    
+
+    <div class="main">
+     
+      <header id="header">
+        <div id="header-img">
+          <img src=https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png width="300px">
+          </div>
+          <nav id="nav-bar">
+     <ul>
+         <li><a href="#Features" class="nav-link">Features</a></li>
+         <li><a href="#How It Works" class="nav-link">How it Works</a></li>
+         <li><a href="#Pricing" class="nav-link">Pricing</a></li>
+     </ul>
+  </nav>
+     </header>
+    
+  
+     <div class="container-subscriber">
+       <form action="https://www.freecodecamp.com/email-submit">
+  
+              <h1>Handcrafted, home-made masterpieces</h1>
+              <input type="email" class="email" placeholder="Enter your email address" required><br>
+              <input type="submit" id="submit" value="GET STARTED">
+  
+       </form>
+  
+    </div>
+  
+    <section id="Features" class="container">
+    
+     <div id="containerBox">
+     <div class="container-icon">
+         <i class="fa fa-3x fa-fire"></i>
+         <img src="https://www.iconsdb.com/icons/preview/orange/fire-2-xxl.png" alt="fire">
+   
+     </div>
+     <div class="container-text">
+         <h1>Premium Materials</h1>
+         <p>Our trombones use the shiniest brass which is sourced locally.
+              This will increase the longevity of your purchase.</p>
+         
+     </div>
+   
+      </div>
+  
+          <div id="containerBox">
+         <div class="container-icon">
```

Comment:
Fix strange indentation.

Reply:
- [ ] Please include an explanation, a "why", for each requested change. For example, This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
This request lacks a "why": *Why don't you use an article tag here instead of a div?*. 🙃😁

- [Optional] Consider using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.😁